database_cleanup/models/create_indexes.py

```diff
@@ -1,82 +1,80 @@
 # Copyright 2017 Therp BV <http://therp.nl>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 # pylint: disable=consider-merging-classes-inherited
+from odoo import fields, models
+
 from ..identifier_adapter import IdentifierAdapter
-from odoo import api, fields, models
 
 
 class CreateIndexesLine(models.TransientModel):
-    _inherit = 'cleanup.purge.line'
-    _name = 'cleanup.create_indexes.line'
+    _inherit = "cleanup.purge.line"
+    _name = "cleanup.create_indexes.line"
+    _description = "Create Indexes Wizard Lines"
 
-    purged = fields.Boolean('Created')
-    wizard_id = fields.Many2one('cleanup.create_indexes.wizard')
-    field_id = fields.Many2one('ir.model.fields', required=True)
+    purged = fields.Boolean("Created")
+    wizard_id = fields.Many2one("cleanup.create_indexes.wizard")
+    field_id = fields.Many2one("ir.model.fields", required=True)
 
-    @api.multi
     def purge(self):
         tables = set()
-        for field in self.mapped('field_id'):
+        for field in self.mapped("field_id"):
             model = self.env[field.model]
-            name = '%s_%s_index' % (model._table, field.name)
+            name = "{}_{}_index".format(model._table, field.name)
```

Comment:
Please don't replace string substitution by `format`. See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#33idioms

Reply:
Was it actually pre-commit that did this?

Reply:
> Was it actually pre-commit that did this?

I guess so. I didn't do that knowingly.

Reply:
I see it's `pyupgrade` from our pre-commit that does that, but only when it's safe so it's alright. I'm asking the lint-masters about it here: https://github.com/OCA/pylint-odoo/issues/302#issuecomment-760754041